packages/maskbook/src/plugins/Wallet/SNSAdaptor/SelectProviderDialog/index.tsx

```diff
@@ -183,7 +188,10 @@ function SelectProviderDialogUI(props: SelectProviderDialogUIProps) {
                     // create a new wallet
                     if (isEnvironment(Environment.ManifestOptions))
                         history.push(`${DashboardRoute.Wallets}?create=${Date.now()}`)
-                    else await Services.Welcome.openOptionsPage(DashboardRoute.Wallets, `create=${Date.now()}`)
+                    else {
+                        // await Services.Welcome.openOptionsPage(DashboardRoute.Wallets, `create=${Date.now()}`)
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/DimensionDev/Maskbook/pull/3631/commits/9d3a0d4cec11f4da01c80875a38011ae4ef0afcc